blivet/formats/fs.py

```diff
@@ -168,6 +173,18 @@ def dict(self):
                   "mountable": self.mountable})
         return d
 
+    @classmethod
```

Comment:
I'm gonna reply to this since it was my idea to make it a class method. The reason is that in many cases we want to know how much space a file system would provide even if it doesn't exist (as an object). Think about things like a list of file systems shown to user together with estimations of how much space they would get. And also in anaconda, we need this estimations early e.g. in disk selection where there are no devices and no file systems, but just a number giving the disk free space size.